sorc/checkout.sh

```diff
@@ -96,13 +102,23 @@ function checkout() {
   return 0
 }
 
-while getopts ":chm:o" option; do
+while getopts ":chgum:o" option; do
   case $option in
     c)
       echo "Recieved -c flag, will delete any existing directories and start clean"
       export CLEAN="YES"
       ;;
+    g)
```

Comment:
Similar to Rahul's question. Can we just keep `CHECKOUT_GSI="YES"` for `-g` and `CHECKOUT_GDAS="YES"` for `-u` so that `-gu` would allow for both to be built? I could see a use case where we might want both analyses to run on the same model fcst (but only one increment) for testing purposes.

Reply:
This works for me.  I agree with @CoryMartin-NOAA .   I can see developers wanting to run both GSI and UFS based DA from the same g-w installation.  This type of testing will frequently occur as we cross check between the two DA engines.   Performing these tests from the same `HOMEgfs` makes sense.    

I confirmed that @CoryMartin-NOAA 's suggestion works.   After making the suggested changes in a working copy, `checkout.sh -g -u` results in both the GSI and GDASApp repos being cloned.   Is this approach acceptable to the g-w team?